tests/test_repository.py

```diff
@@ -697,6 +775,23 @@ def tearDown(self):
             cwd=WORKING_REPO_DIR,
         )
 
+        all_local_tags = subprocess.run(
+            ["git", "tag", "-l"],
+            stderr=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            check=True,
+            cwd=WORKING_REPO_DIR,
+        ).stdout.strip()
+
+        if len(all_local_tags):
+            subprocess.run(
+                ["git", "tag", "-d", all_local_tags],
+                stderr=subprocess.PIPE,
+                stdout=subprocess.PIPE,
+                check=True,
+                cwd=WORKING_REPO_DIR,
+            )
```

Comment:
Needed to add a few tweaks to ensure the repo was raw now that tag support is here!